src/models/spells/slowmode.spell.ts

```diff
@@ -0,0 +1,51 @@
+import { game } from "../../server";
+import { DEFAULT_PLAYER_SPEED, Player } from "../player";
+import { Spell } from "./spell";
+
+export class SlowModeSpell extends Spell {
+  constructor() {
+    super({
+      id: 1,
+      name: "Slow Mode",
+      cooldown: 30,
+      description: "This spell active the slow mode on the unity player.",
+      duration: 10,
+    });
+  }
+
+  cast(player: Player): void {
+    player.speed = 5;
+    this.active = true;
+    this.currentCooldown = this.cooldown;
+    this.timer = this.duration;
+  }
+
+  update(player: Player): void {
```

Comment:
`update()` method could be a shared method of `Spell` abstract class because there's at the moment no special implementation of the update method for a spell